### DISCUSSION TITLE
Help request for on-device build of vlc

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#9914] [Q&A] [answered]
This is just an opener of a request for help.  I am a fairly advanced user of vlc. This I use on my servers for media streaming. I also have it installed within Termux from the it's pointless tepo. However, this is poorly supported, for example, the http interface is now not supported. I am trying to do my own build on phone. I am getting errors. Would some kind person here be willing to assist me please?

**Comment by xtkoba** [comment]
Duplicate of (or continuation from) #9323.

Note that now `vlc` has been packaged in our official repo. Unfortunately I am not familiar with on-device building and the `build.sh` (which I wrote) may not work.

**Reply by macerg** [reply]
I have installed the version from the official repo but this version (vlc/stable 3.0.17.3)  does not support the http interface. I have had to revert to 3.0.14
from it's-pointless repo.

**Reply by xtkoba** [reply]
Is it just the matter of configuration, or is the support dropped in the newer version? If the former, then please open an issue to request for enabling it. If the latter, then we cannot do anything about that.

**Reply by xtkoba** [reply]
Ah, maybe we only have to enable Lua support:

```sh
  if test "x${have_lua}" != "xyes" ;  then
      AC_MSG_ERROR([Could not find lua. Lua is needed for some interfaces (rc, telnet, http) as well as many other custom scripts. Use --disable-lua to ignore this error.])
  fi
```

(quoted from `configure.ac`.)

**Reply by macerg** [reply]
Yes, I think that is it. Would be really nice to have the QT interface too for x11.

**Comment by xtkoba** [comment, marked as answer]
I will take care of this in PR #9917.
